### ISSUE TITLE
Access denied to app owner?

### ISSUE THREAD

**Issue by vizzah** [binaryage/drydrop#12]
I am getting 'Access denied' for the app owner.
I've invited another 'admin' and then can log-in.

Does this check:

if not users.is_current_user_admin():

includes a check for an app owner??

**Comment by darwin** [comment]
Yes, it should include anyone who has access to the Admin Console of GAE:
http://code.google.com/appengine/docs/python/users/adminusers.html
